man/roxytypes/meta.R

```diff
@@ -0,0 +1,5 @@
+list(
+  format = function(x, name, type, default, description, ...) {
+    glue::glue("(`{type}`)\\cr {description}")
```

Comment:
This is how you can define a custom format (including the `\\cr` which isn't in there by default). After looking through the man files, I realized it actually should be:

```suggestion
    glue::glue("(\\code{{{type}}})\\cr {description}")
```

Or I need to parse the whole thing as markdown, not just the `description` part (probably more user friendly this way).